cwl/workflows/parts/2_drep/2.cwl

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env cwl-runner
```

Comment:
"2.cwl" will make it tricky to debug, toil / cwltool use the file name something on the logs.

Maybe drep_2.cwl?